src/generated/resources/data/forge/tags/worldgen/biome/is_plateau.json

```diff
@@ -0,0 +1,8 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:savanna_plateau",
+    "minecraft:wooded_badlands",
+    "minecraft:meadow"
```

Comment:
As for meadow here, should this be removed from plateau since it is in the plains tag already? it's tougher as meadows are high flat land which signals a plateau but visually, they are very similar to a plains.